godotproject/Player.tscn

```diff
@@ -456,15 +456,8 @@ parameters/swing/blend_position = Vector2( 0, 0 )
 parameters/tongue_launch/blend_position = Vector2( 0, 0 )
 parameters/tongue_retract/blend_position = Vector2( 0, 0 )
 
-[node name="Position2D" type="Position2D" parent="."]
-position = Vector2( 0.660875, 15.8131 )
-
-[node name="SwingPos" type="Position2D" parent="."]
-position = Vector2( 12.5368, -0.573662 )
-
 [node name="Line2D" type="Line2D" parent="."]
-position = Vector2( -23.8093, -2.85048 )
-points = PoolVector2Array( 24.1654, 17.9698, 312.552, 18.1135 )
+points = PoolVector2Array( 0, 0, 15, 0 )
```

Comment:
The position and points here were adjusted based on the frog's sprite being moved